src/cli/agreements.ts

```diff
@@ -13,24 +13,44 @@ import { JobState } from '../definitions'
 const logger = loggingFactory('cli:agreements')
 
 type AgreementWithJobs = Agreement & { jobs: JobModel[] }
+type AgreementFilters = { status?: FilterStatus, pinningStatus?: FilterPinningStatus }
+enum FilterStatus { active = 'active', inactive = 'inactive '}
+enum FilterPinningStatus { pinned = 'pinned', notPinned = 'not-pinned' }
```

Comment:
Instead of this custom Enum, please use directly the `JobState` enum and support all its values. You are printing the information of this enum but you don't allow to filter based on it.

Reply:
hmm..., should we then allow to have multiple filter options. ex: `--pinningStatus pinned --pinningStatus backoff --pinningStatus created`?

Reply:
Hmmm, I guess it would be good, but I don't think it is really crucial, so if it would be easy to make then yes, otherwise won't spend much time on it. It is more important to filter based on all values.

Reply:
Clear 👍